src/orders.jl

```diff
@@ -1,12 +1,23 @@
-function create_order_params(;symbol, side, qty=NaN, notional=NaN, time_in_force, type, extended_hours=false, client_order_id="none")
-  
-  validate_order(side, time_in_force, qty, notional)
-
-  params = Dict("symbol" => symbol,
-  "side" => side,
-  "time_in_force" => time_in_force,
-  "type" => type,
-  "extended_hours" => extended_hours)
+function create_order_params(;symbol, side, qty=NaN, notional=NaN, time_in_force, type, extended_hours=false, client_order_id="none", order_class=nothing)
+
+  validate_order(side, time_in_force, qty, notional, order_class, type)
+
+  if isnothing(order_class)
+    params = Dict("symbol" => symbol,
+    "side" => side,
+    "time_in_force" => time_in_force,
+    "type" => type,
+    "extended_hours" => extended_hours)
+  elseif !isnothing(order_class)
+    params = Dict("symbol" => symbol,
+    "side" => side,
+    "time_in_force" => time_in_force,
+    "type" => type,
+    "extended_hours" => extended_hours,
+    "order_class" => order_class,
+    "take_profit" => Dict("limit_price" => tp_limit_price),
+    "stop_loss" => Dict("stop_price" => sl_stop_price, "limit_price" =>  sl_limit_price))
```

Comment:
Error is thrown as these aren't passed into the function. 
They don't need to be passed into the function, the tp/sl params are used in the bracket function. 
Also duplication of code

Reply:
i will check - intention was if order_class=nothing it was the normal orders up to this point and if it was specified then would revert to the ifelse statement in the function. order_class="bracket" along with an assetion in utils :  @assert order_class in ["simple", "bracket", "oco ", "oto"] "valid order class :: simple, bracket, oco, oto". i did this so that we could do a dict within dict for the tp,sl of these order types. i will verify what happened here.

Reply:
k strategy wise - should we leave function create_order_params as is - break out order class and their associated attributes tp, sl and have them only apart of those functions.

Reply:
![image](https://github.com/dm13450/AlpacaMarkets.jl/assets/23480124/f5a76c79-8ca0-4164-8b2a-4f10548ef9a8)